src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private final Map<Fruit, Integer> testMap = new HashMap<>();
+    private ReportGenerator reportGenerator = new ReportGeneratorImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        Storage.clear();
+    }
+
+    @Test
+    void getReport_StorageWithFruits_Ok() throws NoSuchFieldException {

```

Comment:
The test method doesn't throw NoSuchFieldException, so it shouldn't be in the method signature.

Reply:
ok, done